qa/rpc-tests/mvf-bu-retarget.py

```diff
@@ -15,7 +15,7 @@
 import decimal
 
 # period (in blocks) from fork activation until retargeting returns to normal
-HARDFORK_RETARGET_BLOCKS = 180*144
+HARDFORK_RETARGET_BLOCKS = 90*144
```

Comment:
please add `MVF-BU TODO:` comment here to remind us that we have to revert this later